<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="NSDMailSender.qrc">
    <normaloff>:/images/NSD.ico</normaloff>:/images/NSD.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QGroupBox" name="grp_Sender">
      <property name="title">
       <string> From :</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_Pass">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_Port">
         <property name="text">
          <string>SMTP Port : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_Port</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_Server">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>smtp.gmail.com</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_Server">
         <property name="text">
          <string>Server : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_Server</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="le_Port">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>465</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chk_Encrypted">
           <property name="text">
            <string>Encrypted</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_Pass">
         <property name="text">
          <string>Password : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_Pass</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_User">
         <property name="text">
          <string>User : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_User</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_User">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Load_Mail_Sender">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Load Mail Sender Specification</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="NSDMailSender.qrc">
             <normaloff>:/images/open.ico</normaloff>:/images/open.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Save_Mail_Sender">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save Mail Sender Specification</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="NSDMailSender.qrc">
             <normaloff>:/images/save.ico</normaloff>:/images/save.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="grp_Ge">
      <property name="title">
       <string> To : </string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="QCheckBox" name="chk_Payslip">
         <property name="text">
          <string>Payroll Payslip PDF Files</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+P</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_Subject">
         <property name="text">
          <string>Subject : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_Subject</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QComboBox" name="cbb_Attachments">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="readOnly" stdset="0">
            <bool>false</bool>
           </property>
           <property name="clearButtonEnabled" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Browse">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>select file(s) [ctrl+B]</string>
           </property>
           <property name="text">
            <string>_</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+B</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbl_Attachments">
         <property name="text">
          <string>Attachments : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>cbb_Attachments</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_Recipient">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_Subject">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_Recipient">
         <property name="text">
          <string>Recipient : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>le_Recipient</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_New">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Clear Email Specification</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="NSDMailSender.qrc">
             <normaloff>:/images/new.ico</normaloff>:/images/new.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Open">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Load Email Specification</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="NSDMailSender.qrc">
             <normaloff>:/images/open.ico</normaloff>:/images/open.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Save">
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save Email Specification</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="NSDMailSender.qrc">
             <normaloff>:/images/save.ico</normaloff>:/images/save.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbl_Message">
         <property name="text">
          <string>Message : </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
         </property>
         <property name="buddy">
          <cstring>txt_Message</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QTextEdit" name="txt_Message">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Tahoma'; font-size:12pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Send">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>send email [ctrl+Enter]</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="NSDMailSender.qrc">
          <normaloff>:/images/mail.ico</normaloff>:/images/mail.ico</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Exit">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Ctrl+X</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="NSDMailSender.qrc">
          <normaloff>:/images/close.ico</normaloff>:/images/close.ico</iconset>
        </property>
        <property name="shortcut">
         <string>Ctrl+X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_Wait">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>Please Wait !</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>te_Output</cstring>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QTextEdit" name="te_Output">
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_Sender">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Mail Sender Specification</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 255);</string>
      </property>
      <property name="text">
       <string>Sender</string>
      </property>
      <property name="icon">
       <iconset resource="NSDMailSender.qrc">
        <normaloff>:/images/mail.ico</normaloff>:/images/mail.ico</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnu_Main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_New"/>
    <addaction name="act_Open"/>
    <addaction name="act_Save"/>
    <addaction name="separator"/>
    <addaction name="act_Load_Mail_Sender"/>
    <addaction name="act_Save_Mail_Sender"/>
    <addaction name="separator"/>
    <addaction name="act_Exit"/>
   </widget>
   <widget class="QMenu" name="mnu_About">
    <property name="title">
     <string>About</string>
    </property>
    <widget class="QMenu" name="menuHelp">
     <property name="title">
      <string>Help</string>
     </property>
     <addaction name="actionShortcuts"/>
     <addaction name="actionManual"/>
    </widget>
    <addaction name="act_About"/>
    <addaction name="menuHelp"/>
   </widget>
   <addaction name="mnu_File"/>
   <addaction name="mnu_About"/>
  </widget>
  <widget class="QToolBar" name="tlb_Main">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="stb_Main"/>
  <action name="act_New">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="act_Open">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_Save">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_Save_Mail_Sender">
   <property name="text">
    <string>Save Mail Sender</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="act_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="act_Load_Mail_Sender">
   <property name="text">
    <string>Load Mail Sender</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="act_About">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionShortcuts">
   <property name="text">
    <string>Shortcuts</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_Server</tabstop>
  <tabstop>le_Port</tabstop>
  <tabstop>chk_Encrypted</tabstop>
  <tabstop>le_User</tabstop>
  <tabstop>le_Pass</tabstop>
  <tabstop>le_Recipient</tabstop>
  <tabstop>le_Subject</tabstop>
  <tabstop>cbb_Attachments</tabstop>
  <tabstop>btn_Browse</tabstop>
  <tabstop>txt_Message</tabstop>
  <tabstop>btn_Load_Mail_Sender</tabstop>
  <tabstop>btn_Save_Mail_Sender</tabstop>
  <tabstop>btn_Open</tabstop>
  <tabstop>btn_Save</tabstop>
  <tabstop>chk_Payslip</tabstop>
  <tabstop>te_Output</tabstop>
 </tabstops>
 <resources>
  <include location="NSDMailSender.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_Sender</sender>
   <signal>clicked(bool)</signal>
   <receiver>grp_Sender</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Save</sender>
   <signal>clicked()</signal>
   <receiver>act_Save</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Open</sender>
   <signal>clicked()</signal>
   <receiver>act_Open</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Load_Mail_Sender</sender>
   <signal>clicked()</signal>
   <receiver>act_Load_Mail_Sender</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_New</sender>
   <signal>clicked()</signal>
   <receiver>act_New</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Save_Mail_Sender</sender>
   <signal>clicked()</signal>
   <receiver>act_Save_Mail_Sender</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
